controller/array_action/ds8k_rest_client.py

```diff
@@ -0,0 +1,253 @@
+from pyds8k.client.ds8k.v1.client import Client
+from pyds8k.exceptions import NotFound
+from controller.common.csi_logger import get_stdout_logger
+
+logger = get_stdout_logger()
+
+
+def normalize_response(func):
```

Comment:
import `dictionarize` instead

Reply:
I won't import anything from SCClient. But I agree that it is not the best way, I will consider to implement method like as_list, as_dict in pyds8k, so that we can get rid of the decorator.

Reply:
OK, so I'm leaving this conversation unresolved for not

Reply:
@27149chen now that the mediator doesn't use the representation dict, can we resolve this?